post-process-pe.c

```diff
@@ -360,6 +364,34 @@ set_dll_characteristics(PE_COFF_LOADER_IMAGE_CONTEXT *ctx)
 	}
 }
 
+static void
+ms_validation(PE_COFF_LOADER_IMAGE_CONTEXT *ctx)
+{
+	EFI_IMAGE_SECTION_HEADER *Section;
+	int i;
+
+	debug(INFO, "%14s: %s\n","NX-Compat-Flag",
```

Comment:
What's with all this `"%14s:` stuff? Why not just `"NX-Compat-Flag: %s\n"`? Same with all the cases below it.